tests/test_create_checkout.py

```diff
@@ -0,0 +1,39 @@
+from typing import TYPE_CHECKING
+from unittest.mock import Mock, patch
+
+import stripe
+
+if TYPE_CHECKING:
+    from fastapi.testclient import TestClient
+
+
+@patch("stripe.checkout.Session.create")
+def test_create_checkout_should_return_url(mock_stripe: Mock, client: "TestClient") -> None:
```

Comment:
test_should_create_new_checkout_session <- we test behave, not what will be returned